src/common/interfaces/response.ts

```diff
@@ -1,7 +1,6 @@
-import { HttpStatus } from "@nestjs/common";
 
 export interface IResponse {
-  data: any;
+  data?: any;
```

Comment:
why any

Reply:
@agrawalchiragdeqode sir,   because data object can have any information , accessToken , entity data , depending upon use case and ApI service being called.

Reply:
OR m kardo different formats